Makefile

```diff
@@ -39,10 +39,10 @@ install-all: install install-ostree-hooks
 	install -D -m 0755 target/release/tests-integration $(DESTDIR)$(prefix)/bin/bootc-integration-tests 
 
 bin-archive: all
-	$(MAKE) install DESTDIR=tmp-install && $(TAR_REPRODUCIBLE) --zstd -C tmp-install -cf target/bootc.tar.zst . && rm tmp-install -rf
```

Comment:
-zstd unsupported on `quay.io/centos/centos:stream9`

Reply:
Wait that's definitely not true. Do you have a really old image? Builds don't pull by default so unfortunately you need to explicitly e.g. `podman pull quay.io/centos-bootc/centos-bootc:stream9` or so.

Reply:
I don't believe I have an old image